app/backend/src/couchers/servicers/events.py

```diff
@@ -978,9 +978,23 @@ def SetEventAttendance(self, request, context, session):
         return event_to_pb(session, occurrence, context)
 
     def ListMyEvents(self, request, context, session):
+        # Check which field in the 'pagination' oneof is set
+        pagination_type = request.WhichOneof("pagination")
         page_size = min(MAX_PAGINATION_LENGTH, request.page_size or MAX_PAGINATION_LENGTH)
-        # the page token is a unix timestamp of where we left off
-        page_token = dt_from_millis(int(request.page_token)) if request.page_token else now()
+
+        if pagination_type == "page_token":
+            # the page token is a unix timestamp of where we left off
+            page_token = dt_from_millis(int(request.page_token))
+            page_number = 1
```

Comment:
I don't think these `page_number`s are used now outside the `pagination_type == "page_number"` case

Reply:
You are right, apologies. I realize I ran the tests before regenerating the protos and assumed the error was this.